python/python_helper_sympy_test.py

```diff
@@ -37,7 +36,7 @@ class TestSympy:
         "acosh(m)",
     ]
 
-    EXPR_LIST: List = [
+    EXPR_LIST: list = [
```

Comment:
micro-nit: is declaring this type actually necessary? Seems like this is something that should be easily inferred by type checkers.

Reply:
I think in this instance it isn't, but in general it's a good idea to be _very_ explicit with annotations in Python. Most of the inferences that get made can be overriden really easily by calling a function incorrectly.

I don't want to make a ton of changes to this file here though, it's already being reworked in https://github.com/PrairieLearn/PrairieLearn/pull/7020 and the version there is much easier to read.